sentry-ruby/lib/sentry/hub.rb

```diff
@@ -92,7 +93,9 @@ def start_transaction(transaction: nil, custom_sampling_context: {}, **options)
       transaction
     end
 
-    def with_child_span(**attributes, &block)
+    def with_child_span(instrumenter: :sentry, **attributes, &block)
```

Comment:
Could users use both instrumenters (`sentry` and `otel`) at the same time? Or it's simpler like: if users chose `otel`, they'll give up `sentry` tracing completely.

Reply:
for now, it's one or the other. I don't know what direction we will take on this on the future, depends on a lot of parameters like adoption/future of SDKs etc